src/main/java/org/rev317/min/api/methods/Walking.java

```diff
@@ -16,7 +20,13 @@
      * @param to   Tile to Walk To.
      */
     public static void walkTo(Tile from, Tile to) {
-        Loader.getClient().walkTo(0, 0, 0, 0, from.getRegionY(), 0, 0, to.getRegionY(), from.getRegionX(), true, to.getRegionX());
+        if (Context.getInstance().getServerProviderInfo().getServerName().toLowerCase().contentEquals("pkhonor")) {
+            RefClass refClass = new RefClass(Loader.getClient());
```

Comment:
This doesn't seem the appropriate way, simply add a new accessor for the pkhonor walking